src/styles/_catalog.scss

```diff
@@ -1,22 +1,25 @@
 .catalog {
-  display: flex;
+  display: grid;
+  grid-template: 300px / repeat(4, 200px);
   flex-wrap: wrap;
```

Comment:
We don't need flex-wrap, the display is grid

Reply:
agree, delete redundant property